prometheus/common/alert-rules.d/crunchy-alert-rules-etcd.yml.example

```
@@ -10,11 +10,58 @@ groups:
 
 ########## ETCD EXPORTER RULES ##########
 
+#### IMPORTANT NOTE: These alerts only support monitoring a single etcd cluster per promemtheus instance ####
+
+# Ensure there is at least one etcd node reporting metrics
+  - alert: ECTDAbsent
```

```suggestion
  - alert: ETCDAbsent
```
Note: not sure if there are other things that rely on the wrong name, so that might need to be updated to match as well.

---

No, the name is only used here. Thanks!